css/selectors/is.json

```diff
@@ -1,4 +1,4 @@
-{
+ {
```

Comment:
Spurious change?

(If you really want to add some random characters here, I'm especially fond of `:wq`.)